src/tools/rbd/action/Journal.cc

```diff
@@ -973,6 +1062,10 @@ Shell::Action action_reset(
   {"journal", "reset"}, {}, "Reset image journal.", "",
   &get_reset_arguments, &execute_reset);
 
+Shell::Action action_disconnect(
+  {"journal", "disconnect"}, {}, "Flag image journal client disconnected.", "",
```

Comment:
How about "journal client disconnect"? Since I can also see the debugging need for "journal client remove"